lib/improver/tests/utilities/test_save.py

```diff
@@ -45,28 +45,23 @@
 from improver.utilities.save import save_netcdf
 from improver.utilities.save import append_metadata_cube
 from improver.utilities.save import order_cell_methods
-from improver.tests.ensemble_calibration.ensemble_calibration. \
-    helper_functions import set_up_cube
+from improver.tests.set_up_test_cubes import set_up_variable_cube
 
 
 def set_up_test_cube():
     """ Set up a temperature cube with additional global attributes. """
-    data = (
-        np.linspace(-45.0, 45.0, 9, dtype=np.float32).reshape(1, 1, 3, 3) +
-        273.15
-    )
-    cube = set_up_cube(data, 'air_temperature', 'K', realizations=([0]))
-    cube.attributes['test_attribute'] = np.arange(12)
-    # Desired attributes that will be global in netCDF file
-    cube.attributes['um_version'] = '10.4'
-    cube.attributes['mosg__grid_type'] = 'standard'
-    cube.attributes['mosg__model_configuration'] = 'uk_ens'
-    cube.attributes['mosg__grid_domain'] = 'uk_extended'
-    cube.attributes['mosg__grid_version'] = '1.2.0'
-    cube.attributes['source'] = 'Met Office Unified Model'
-    cube.attributes['Conventions'] = 'CF-1.5'
-    cube.attributes['institution'] = 'Met Office'
-    cube.attributes['history'] = ''
+    data = np.linspace(
+        -45.0, 45.0, 9, dtype=np.float32).reshape(1, 3, 3) + 273.15
+
+    attributes = {
```

Comment:
Why have you chosen not to include a mosg attribute?

Reply:
I've included these using the `standard_grid_metadata` keyword argument to the `set_up_variable_cube` function.  The test cube has all four `mosg__` attributes as before.

Reply:
Ok, sorry I missed that. That's fine then